.github/workflows/link-check-deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: Check new links against deployment
```

Comment:
Do we want a daily link check as well? 🤔

Reply:
Good point! Added that in.